.travis.yml

```diff
@@ -4,13 +4,9 @@
 language: generic
 
 os: osx
+osx_image: beta-xcode6.1
 
 env:
-  matrix:
-    
-    - CONDA_PY=27
-    - CONDA_PY=34
-    - CONDA_PY=35
```

Comment:
So this re-rendering seems off. We have seen this issue before<sup>1</sup>. The fix was to get everything from the conda-forge channel in `root`. Maybe try that either in your current Miniconda install or a new one and see if that doesn't work.


1 - https://github.com/conda-forge/conda-smithy/issues/223